tests/stubtest_third_party.py

```diff
@@ -126,15 +131,15 @@ def run_stubtest(dist: Path, *, verbose: bool = False, specified_stubs_only: boo
             print_success_msg()
 
     if verbose:
-        print_commands(dist, pip_cmd, stubtest_cmd)
+        print_commands(dist, pip_cmd, stubtest_cmd, mypypath)
 
     return True
 
 
-def print_commands(dist: Path, pip_cmd: list[str], stubtest_cmd: list[str]) -> None:
+def print_commands(dist: Path, pip_cmd: list[str], stubtest_cmd: list[str], mypypath: str) -> None:
```

Comment:
@AlexWaygood The `dist` argument after this change has been left unused. Was that intentional ?

Reply:
Nope! It makes sense to get rid of it :D